tests/e2e/specs/SmokeTest.spec.ts

```diff
@@ -46,7 +49,18 @@ suite('The SmokeTest userstory', function (): void {
 		});
 		test('Check a project folder has been created', async function (): Promise<void> {
 			const projectName: string = FACTORY_TEST_CONSTANTS.TS_SELENIUM_PROJECT_NAME || StringUtil.getProjectNameFromGitUrl(factoryUrl);
-			projectSection = await new SideBarView().getContent().getSection(projectName);
+			for (let attempts: number = 0; attempts < TIMEOUT_CONSTANTS.TS_SELENIUM_DEFAULT_ATTEMPTS; attempts++) {
+				try {
+					projectSection = await new SideBarView().getContent().getSection(projectName);
+				} catch (err) {
+					if (attempts === TIMEOUT_CONSTANTS.TS_SELENIUM_DEFAULT_ATTEMPTS) {
```

Comment:
what is the default number of attempts?

Reply:
problem was elsewhere, this particular logic was removed